frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -562,19 +658,37 @@ export function NeuronSegmentationModal({ handleClose }: Props) {
       }}
       description={
         <>
-          <p>
-            Start an AI background job that automatically detects and segments all neurons in this
-            dataset. The AI will create a copy of this dataset containing the new neuron
-            segmentation.
-          </p>
-          <p>
-            <b>
-              Note that this feature is still experimental and processing can take a long time.
-              Thus, we suggest to use a small bounding box and not the full dataset extent. The
-              neuron detection currently only works with EM data. The best resolution for the
-              process will be chosen automatically.
-            </b>
-          </p>
+          <Space direction="vertical" size="middle">
+            <Row>
+              This job will automatically detect and segment all neurons in this dataset. The AI
+              will create a copy of this dataset containing the new neuron segmentation.
+            </Row>
+            <Row style={{ display: "grid", marginBottom: 16 }}>
+              <div
+                style={{
+                  gridColumnStart: 1,
+                  gridColumnEnd: 2,
+                  gridRowStart: 1,
+                  gridRowEnd: 3,
+                  margin: "auto",
+                }}
+              >
+                <ExclamationCircleOutlined style={{ color: "#ffda33", fontSize: 22 }} />
+              </div>
+              <div
+                style={{
+                  gridColumnStart: 2,
+                  gridColumnEnd: 3,
+                  gridRowStart: 1,
+                  gridRowEnd: 3,
+                  marginLeft: 4,
+                }}
+              >
+                Please note that this feature is experimental and currently only works with electron
+                microscopy data.
+              </div>
+            </Row>
```

Comment:
You could have a look at [Antd Alert](https://ant.design/components/alert)s which we usually use for this purpose.